data-sdk/src/main/java/com/logicmonitor/sdk/data/api/Logs.java

```diff
@@ -148,7 +148,7 @@ public Optional<ApiResponse> sendLogs(
       final Map<String, String> resourceId,
       final Map<String, String> metadata)
       throws IOException, ApiException {
-    final String timeStamp = String.valueOf(Instant.now().getEpochSecond());
```

Comment:
why are we using the current time? it should be passed from a user if not passed we should use the current time.

Reply:
made the changes to accept the timestamp from user